docs/snippets/modules/chains/popular/summarize.mdx

```diff
@@ -283,6 +283,27 @@ map_reduce.run(input_text=code, question="Which function has a better time compl
 
 </CodeOutputBlock>
 
+## `MapReduceChain` - Adjusting for different context sizes and `token_max`
+
+The ReduceDocumentsChain uses a `token_max` arg to decide the maximum limit of tokens the reduce step can take in. The MapReduceChain effectively chunks up and condenses inputs until they drop below the `token_max` threshold.
+Once below this `token_max`, the chain will run the reduce step. By default `token_max` is set to `3000` tokens.
+
+You may want to change this when dealing with models with larger/smaller context sizes. This arg also acts as a sliding scale between how much you want to condense inputs before passing them to the reduce step vs how many tokens you want to leave for generation in the reduce step.
+Eg: If you have 16k tokens and you set the `token_max` to 5k, your inputs will be condensed until they together are at most 5k tokens, but leave 11k tokens for the final output generation.
+
+Do not confuse this with `max_tokens` which is the maximum tokens you can get from a model's generation.
+
+You can set the `token_max` like so when creating a `ReduceDocumentsChain`:
```

Comment:
don't think ReduceDocumentsChain is mentioned anywhere before this? should we update the custom map reduce section first, so that this is better contextualized?

Reply:
@baskaryan I'll get to that and update the PR.